### PR TITLE
feat(unlock-protocol-com) - change header component

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

Switch to new `HeaderNav` component

Screenshot 2022-12-12 at 13 50 04

Screenshot 2022-12-12 at 13 50 11

Screenshot 2022-12-12 at 13 50 16

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by searchableguy** [comment]
State of unlock is not alive. You can check the PR for where the link will be and add a commented out link.

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 1]
```diff
@@ -1,15 +1,185 @@
...
+              label: 'Unlock Smart Contracts',
+              url: 'https://docs.unlock-protocol.com/core-protocol/',
+            },
+            {
+              label: 'Deploying Locks',
+              url: 'https://docs.unlock-protocol.com/core-protocol/public-lock/deploying-locks/',
+            },
+            {
+              label: 'Hooks',
+              url: 'https://docs.unlock-protocol.com/core-protocol/smart-contracts-api/archive/hooks', //todo: fix link
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:29]:
```suggestion
              url: 'https://docs.unlock-protocol.com/core-protocol/public-lock/hooks'
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 2]
```diff
@@ -1,15 +1,185 @@
...
+            },
+          ],
+        },
+        {
+          title: 'Tools',
+          options: [
+            {
+              label: 'Building token gated applications',
+              url: 'https://docs.unlock-protocol.com/tutorials/building-token-gated-applications/',
+            },
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:39]:
```suggestion
            {
              label: 'Unlock-js',
              url: 'https://docs.unlock-protocol.com/tools/unlock.js',
            },
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 3]
```diff
@@ -1,15 +1,185 @@
...
+            {
+              label: 'Sign in with Ethereum',
+              url: 'https://docs.unlock-protocol.com/tools/sign-in-with-ethereum/',
+            },
+            {
+              label: 'Webhooks',
+              url: 'https://docs.unlock-protocol.com/tools/locksmith/webhooks/',
+            },
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:47]:
```suggestion
```

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 4]
```diff
@@ -1,15 +1,181 @@
...
+              url: 'https://unlock-protocol.com/upcoming-events',
+            },
+            {
+              label: 'Grants Program',
+              url: 'https://unlock-protocol.com/grants',
+            },
+            {
+              label: 'Got Questions?',
+              url: '', // todo: send to email? discord? form?
+            },
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:158]: I say we remove this for now?

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 5]
```diff
@@ -1,15 +1,181 @@
...
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:158]:
```suggestion
```

**Comment by julien51** [comment]
@kalidiagne Can you check all the links to make sure we dpon't have the same issue as the footer please?

**Comment by kalidiagne** [comment]
> @kalidiagne Can you check all the links to make sure we dpon't have the same issue as the footer please?

Fix already present now with rebase

**Comment by kalidiagne** [comment]
### Desktop 

Screenshot 2022-12-20 at 12 26 20

Screenshot 2022-12-20 at 12 26 25

Screenshot 2022-12-20 at 12 26 35

### Mobile

**Review by julien51 — APPROVED** [review]
Ok let's ship it please!

**Pull request merged**